server/src/main/java/com/objectcomputing/checkins/services/guild/GuildRepository.java

```diff
@@ -23,13 +23,16 @@
             "FROM guilds guild " +
             "WHERE (:name IS NULL OR guild.name LIKE :name) " +
             "AND (:memberid IS NULL OR guild.id IN " +
-                "(SELECT guildid FROM guildmembers gm " +
```

Comment:
I would argue the tabbing here is useful as it shows that the subquery being at the SQL version of a lower scope than the rest of the query.

Reply:
Intellij must have changed that